plone/dexterity/filerepresentation.py

```diff
@@ -838,7 +836,10 @@ def write(self, data):
         if self._closed:
             raise ValueError("File is closed")
         self._written += len(data)
-        self._parser.feed(data)
+        if isinstance(data, bytes):
```

Comment:
@Rotonen It must be this one. I’ll try to fix asap I am on kbd.

Reply:
I already have a couple of PRs incoming:

https://github.com/plone/plone.dexterity/pull/104
https://github.com/plone/plone.rfc822/pull/8